### ISSUE TITLE
[Moderate] Label does not convey required format

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#70]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87560 |
| **Checkpoint** | 3.3 Input Assistance › Missing Instructions (3.3.2.b) |
| **Impact** | Moderate |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Account / Change Password |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/users/my_account> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/05/26 at 4:40 pm |
| **Unit Type** | Page |
| **Issue Type** | Best Practice |
| **Reported By** | a3c.access@gmail.com |

## Description

The input requires a specific format but the label or instruction does not specify this.
On the Change Password page, there is no information regarding the length of the password. Since the password requires 12 or more characters, it would be helpful to provide this information at he top of the password dialog.

## Recommendation

How to fix:
Provide the password requirements as part of the label.

Background: 
Filling out forms correctly can be one of the more time consuming and frustrating online user experiences, and it can be even more challenging for people with disabilities. The single easiest way to prevent user errors is by providing clear labels and instructions that are available to everyone at all times. Labels and instructions should:
1) be clear and informative
2) disclose any constraints such as required data formats or ranges, and
3) identify required fields.
Well-designed labels and instructions provide enough information so people can fill out forms without undue confusion or errors.

## Source Code

```
<input class="input" type="password" id="NewPassword" name="NewPassword" value="" placeholder="New Password" autocomplete="new-password" required="">
```

## Screenshots

![02_missing instructions](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87560/02_missing_instructions.png)
